packages/forms/src/styled/text/StyledTextFauxInput.ts

```diff
@@ -14,14 +14,12 @@ const COMPONENT_ID = 'forms.faux_input';
 interface IStyledTextFauxInputProps extends IStyledTextInputProps {
   mediaLayout?: boolean;
   isDisabled?: boolean;
-  isFocused?: boolean;
 }
 
 export const StyledTextFauxInput = styled(StyledTextInput).attrs<IStyledTextFauxInputProps>(
   props => ({
     as: 'div',
     'aria-disabled': props.isDisabled,
-    'data-garden-focus-visible': props.isFocused,
```

Comment:
Since this value is controlled by the `useFocusVisible()` hook it can be added/removed outside of the render cycle in React.

This value would apply initially, but then be removed based on key/mouse interactions. I've added a `isFocused` prop to the `StyledTextInput` element which is inherited here.